kotest-property/src/commonMain/kotlin/io/kotest/property/Gen.kt

```diff
@@ -68,6 +70,11 @@ abstract class Arb<out A> : Gen<A>() {
     */
    abstract fun edgecases(): List<A>
 
+   open fun edges(): EdgeCases<A> =
```

Comment:
i haven't annotated edgecases() as deprecated as i realized this `edges()` can theoretically coexists with the current `edgecases(): List<A>`, e.g. making it internal or some sort. what's the best course of action here @sksamuel ?